src/components/form/form.tsx

```diff
@@ -54,6 +55,10 @@ export const EuiForm: FunctionComponent<EuiFormProps> = ({
   invalidCallout = 'above',
   ...rest
 }) => {
+  const handleFocus = useCallback((node) => {
+    node && node.focus();
```

Comment:
The new optional chaining operator, `?.` cleans this up a little

```suggestion
    node?.focus();
```

verifies `node` is neither undefined nor null before accessing & calling the focus method